instance/tests/models/test_instance.py

```diff
@@ -137,9 +137,10 @@ def test_github_admin_username_list_default(self):
         By default, no admin should be configured
         """
         instance = OpenEdXInstanceFactory()
+        instance.reset_ansible_settings()
```

Comment:
@smarnach The problem if you only update it at provisioning is that https://github.com/open-craft/opencraft/pull/26/files#diff-a8e64e20aeb5487bab76b72ae22e9aefR117 is wrong before the provisioning has started, ie you can't preview the configuration on the Web UI that would be applied to the next provisioning. 

That said, imho the right approach for configurations (beyond the scope of this task) would be to eventually have revisions, each provisioning pointing to a specific configuration revision. That would allow to keep track of the current active configuration, as well as the current, up to date, result of the other fields (ie the preview of the configuration that would be applied to the next provisioning). Since my comment could be better taken care of in that context, that's fine to keep your current approach for now.